priv/static/robots.txt

```diff
@@ -0,0 +1,5 @@
+# See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
```

Comment:
When finalizing the pull request, please do not forget to remove the file `assets/static/robots.txt` to avoid confusion.